app/views/gkr_keyword_comparison/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_tag(gkr_keyword_comparison_path, method: "get") do %>
+    <%= label_tag :concept_id %><br>
+    <%= text_field_tag :Concept_ID %>
+    <br>
+    <br>
+    <%= label_tag :threshold %><br>
+    <%= text_field_tag :Threshold %>
+    <p>
+        <%= submit_tag("Submit") %>
+    </p>
+<% end %>
```

Comment:
I don't think you need this file either, the `show` page should be sufficient

Reply:
You're right, I believe I just forgot to delete that after I was trying to troubleshoot something